demos/browser/webpack.config.js

```diff
@@ -1,20 +1,57 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+/**
+ * See also webpack.config.hot.js, which is a fork of this file. Please make
+ * changes in both places if applicable.
+ */
+
 /* eslint-disable */
-var webpack = require('webpack');
-var HtmlWebpackPlugin = require('html-webpack-plugin');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CspHtmlWebpackPlugin = require('csp-html-webpack-plugin');
 const InlineChunkHtmlPlugin = require('react-dev-utils/InlineChunkHtmlPlugin');
 /* eslint-enable */
 
+/**
+ * This is exactly what we document in the CSP guide.
+ */
+const csp = {
+  'connect-src': "'self' https://*.chime.aws wss://*.chime.aws https://*.amazonaws.com https://*.sdkassets.chime.aws",
+
+  // 'wasm-unsafe-eval' is to allow Amazon Voice Focus to work in Chrome 95+.
+  // Strictly speaking, this should be enough, but the worker cannot compile WebAssembly unless
+  // 'unsafe-eval' is also present.
+  'script-src': "'self' https://*.sdkassets.chime.aws 'unsafe-eval' 'wasm-eval' 'wasm-unsafe-eval'",
+
+  // Script hashes/nonces are not emitted for script-src-elem, so just add unsafe-inline.
+  'script-src-elem': "'self' https://*.sdkassets.chime.aws 'unsafe-inline'",
+  'worker-src': "'self' blob:",
+  'child-src': "'self' blob:",
+};
+
+// Modify our basic CSP to allow several things:
+// 1. Access to gamma assets for testing.
+csp['connect-src'] += ' https://*.sdkassets.gchime.aws';
```

Comment:
I don't think it's super necessary, but with the asset pipeline being used more often and gamma being blocked, it might be nice to be able to test with alpha as well.

Reply:
My hope is that this is sufficiently clear that devs can change it to their local test servers or to other CDNs as needed. This should be enough for routine QA.